parsers/manifest_parser.go

```diff
@@ -389,14 +389,14 @@ func (dm *YAMLParser) ComposeActions(filePath string, actions map[string]Action,
 		if action.Runtime != "" {
 			if utils.CheckExistRuntime(action.Runtime, utils.Rts) {
 				wskaction.Exec.Kind = action.Runtime
-			} else {
-				errStr := wski18n.T("the runtime is not supported by Openwhisk platform.\n")
-				whisk.Debug(whisk.DbgWarn, errStr)
-			}
-		} else {
-			errStr := wski18n.T("wskdeploy has chosen a particular runtime for the action.\n")
-			whisk.Debug(whisk.DbgWarn, errStr)
-		}
+
+			} else if utils.Flags.Strict {
+                wskaction.Exec.Kind = action.Runtime
+            } else {
+                errStr := wski18n.T("wskdeploy has chosen a particular runtime for the action.\n")
```

Comment:
This message should be added into json file as the i18n string. After that, i18n_resources should be regenerated.

Reply:
@houshengbo , thanks for your review, I think this message has already been there, and corresponding resource file are regenerated.
https://github.com/apache/incubator-openwhisk-wskdeploy/blob/master/wski18n/resources/en_US.all.json#L55